### PR TITLE
have d3 version of helloworld created /app/app.js /styles/styles.css and...

### PR DESCRIPTION
**Pull request by Dmeowmixer**
... /views/index.html

### CODE REVIEW

**Comment by keokilee** [comment]
chuck-norris

File: views/index.html [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  <meta charset="UTF-8">
+  <title>Desktop</title>
+</head>
+<body>
+
+
+  <div id="chart">
+
+
+  <script src="http://d3js.org/d3.v3.min.js" charset="utf-8"></script>
```
Comment (keokilee) [inline views/index.html]: Wondering if you want to download this? Bower it, maybe?

**Comment by Dmeowmixer** [comment]
image

File: bower.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
+{
+  "name": "desktop",
+  "version": "0.0.0",
+  "homepage": "https://github.com/Dmeowmixer/desktop",
```
Comment (keokilee) [inline bower.json]: One more thing. Change this homepage to https://github.com/hitraffic/desktop.

**Comment by keokilee** [comment]
Also, the d3 files are still there. Easiest thing to do is to delete the bower_components directory, check that in, and then use bower install to reinstall it.

**Comment by Dmeowmixer** [comment]
:100:

**Comment by keokilee** [comment]
:+1:

**Pull request merged**